src/edit/getEXIFdata.js

```diff
@@ -1,85 +0,0 @@
-L.EXIF = function getEXIFdata(img) {
-  if (Object.keys(EXIF.getAllTags(img)).length !== 0) {
-    console.log(EXIF.getAllTags(img));
-    var GPS = EXIF.getAllTags(img),
-      altitude;
-
-    /* If the lat/lng is available. */
-    if (
-      typeof GPS.GPSLatitude !== "undefined" &&
-      typeof GPS.GPSLongitude !== "undefined"
-    ) {
-      // sadly, encoded in [degrees,minutes,seconds]
```

Comment:
you didn't copy over this code comment I just wanted to confirm if its irrelavent now or could still be useful